### PR TITLE
Track timing for all celery tasks

### PR DESCRIPTION
**Pull request by kumar303**
This should help out a little for investigation of https://github.com/mozilla/olympia/issues/1140

### CODE REVIEW

File: apps/amo/celery.py [hunk 1]
```diff
@@ -40,3 +46,48 @@ def process_failure_signal(exception, traceback, sender, task_id,
                 'kwargs': kwargs
             }
         })
+
+
+@task_prerun.connect
+def start_task_timer(task_id, task, args, kwargs, **kw):
```
Comment (mstriemer) [inline apps/amo/celery.py]: `kwargs` and `kw`? That's a little confusing.

Comment (mstriemer) [inline apps/amo/celery.py]: s/kw/options/?

Comment (kumar303) [inline apps/amo/celery.py]: it's really `task_kwargs`, I'll change it

**Comment by mstriemer** [comment]
Looks good to me if it still works :p

**Pull request merged**